src/contradictory_claims/claims/extract_claims.py

```diff
@@ -0,0 +1,91 @@
+"""Function to extract Claims."""
+
+
+import nltk
+import numpy as np
+import pandas as pd
+import torch
+from allennlp.common.file_utils import cached_path
+from allennlp.models.archival import load_archive
+from allennlp.modules import ConditionalRandomField, TimeDistributed
+from allennlp.predictors import Predictor
+from nltk import sent_tokenize
+from torch.nn import Linear
+
+from .dataset_reader import CrfPubmedRCTReader
+from .predictors import ClaimCrfPredictor
+from .models import DiscourseCrfClassifier  # noqa:F401
```

Comment:
Is this classifier not used? Is this a different classifier that we wanted to play around with as well?